textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/ListField.java

```diff
@@ -7,15 +7,15 @@
 /**
  * Created by kishorenarendran on 25/04/16.
  */
-public class StringListField implements IField {
-    private final List<String> value;
+public class ListField<T> implements IField {
```

Comment:
@kishore-narendran .. Sorry if I am not clear before. I didn't want you to create this class. I wanted you to use ListField once the pull request #77 is merged into master. But for now, the old approach is fine.